src/posts/asking-why-a-practice-is-not-able-to-accommodate-you.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Asking why a practice is not able to accommodate you
+date: '2021-01-10'
+tags:
+  - ada
+  - oral-deaf
+  - deaf
+  - telemedicine
+---
+
+# Context
```

Comment:
All headings in this post need to be bumped up one level. The `title` of this post ("Asking why a practice is not able to accommodate you") becomes a `<h1>`, and the headings in this post are with respect to that title. So this should be `## Context`, `### Who was this written by?`, `### Who was this written for?`, etc.